ui/analyse/css/_analyse.free.scss

```
@@ -15,8 +15,7 @@
       top: 100%;
       text-align: right;
       visibility: hidden;
-
-      @include transition;
+      transition: all 150ms !important;
     }
 
     textarea:focus ~ .action {
```

```suggestion
    textarea:focus ~ .action, .action:focus {
```
i.e. do this ^

---

it might be kind of weird if the button gets focus on click when subsequent activation of it would be redundant.  feel free to step in on this though, i'm not really familiar with it.

---

Hm, you mean that it stays visible after clicking?

Seems kinda difficult to keep it unclickable when hidden, no transition with reduced motion but a nice transition without it, and make it vanish directly after getting clicked 🤔

ig it's easy to at least only keep it visible while hovering
```suggestion
    textarea:focus ~ .action, .action:hover, .action:focus-visible {
```
and the `:focus-visible` keeps it around when using the keyboard. Actually, I wonder how this all is for accessibility and whether using `visibility: hidden` makes that worse than `opacity: 0`. But ig to make it properly accessible the import option should be added to the nvui version of the page.

I guess another option might be to instead just make `preferse-reduced-motion` set `transition-duration` to `1ms` instead of completely disabling them?

---

Yes it stays visible, try it with firefox prefers-reduced-motion on https://schlawg.org

Does it need to be a disappearing button at all?  Maybe it should be visibly enabled when the textarea has focus, disabled otherwise (except nvui), and trap enter key when the textarea is focused like chat.  Give them shift-enter to make a return in the pgn input in case anyone actually wants to enter lines of a pgn by typing them.

---

better yet, screw the button and just trap enter.